front/app/containers/IdeasShowPage/IdeaShowPageTopBar.tsx

```diff
@@ -162,15 +93,7 @@ const IdeaShowPageTopBar = ({
     <Container className={className || ''}>
       <TopBarInner>
         <Left>
-          <GoBackButton onClick={onGoBack}>
-            <GoBackIcon ariaHidden name="arrow-back" />
-            <ScreenReaderOnly>
-              <FormattedMessage {...messages.goBack} />
-            </ScreenReaderOnly>
-          </GoBackButton>
-          <GoBackLabel aria-hidden>
-            <FormattedMessage {...messages.goBack} />
-          </GoBackLabel>
+          <GoBackButton insideModal={insideModal} projectId={projectId} />
```

Comment:
The only thing this PR does is use the GoBackButton from IdeasShow (only 1 button to maintain) and make the insideModal + projectId prop on IdeaShowPageTopbar.

The existing button wasn't very descriptive ("Go back").